style.css

```diff
@@ -60,30 +64,33 @@ body {
 	margin-top: -25rem
 }
 .header-image {
-	width: 80%;
+	width: 60%;
 	height: 40rem;
+	margin: 5rem auto 0 auto;
+
 }
 .button-border {
 	z-index: 1;
 	background-color: #ffffff;
 	display: flex;
 	justify-content: center;
 	position: absolute;
-	width: 10rem;
-	height: 10rem;
+	width: 9rem;
+	height: 9rem;
 	padding: 3rem auto 0 auto;
 	border-radius: 50%;
-	margin-left: -40rem;
-	margin-top: 4rem;
+	margin-left: -25rem;
```

Comment:
What does this do?

Reply:
Positions the play button correctly, in the top left corner

Reply:
Okay let's catch up on Monday about this because there are better ways of positioning elements